tests/testthat/test-get_param.R

```diff
@@ -0,0 +1,48 @@
+test_that("can use configuration file via options", {
```

Comment:
I don't really know what any of this stuff us either, so I guess it's fine

Reply:
`get_param()` gets values from specific fields in a configuration file. It allows you to specify the behavior your want if the field is NULL (e.g. return NULL, throw an error, etc.) and to define the configuration file explicitely or implicitely via `options()`. The interface is awkward because it is a function factory -- it made sense to me in the context of r2dii.utils, but here I feel like I should make it simpler and likely to move it to a thinner version of r2dii.utils.